dll/win32/shell32/CShellLink.cpp

```diff
@@ -2802,6 +2802,29 @@ BOOL CShellLink::OnInitDialog(HWND hwndDlg, HWND hwndFocus, LPARAM lParam)
     if (m_sDescription)
         SetDlgItemTextW(hwndDlg, IDC_SHORTCUT_COMMENT_EDIT, m_sDescription);
 
+    /* Hot key */
+    SendDlgItemMessageW(hwndDlg, IDC_SHORTCUT_KEY_HOTKEY, HKM_SETHOTKEY, m_Header.wHotKey, 0);
+
+    /* Run */
+    const WORD runstrings[] = { IDS_SHORTCUT_RUN_NORMAL, IDS_SHORTCUT_RUN_MIN, IDS_SHORTCUT_RUN_MAX };
+    const BYTE runshowcmd[] = { SW_SHOWNORMAL, SW_SHOWMINNOACTIVE, SW_SHOWMAXIMIZED };
```

Comment:
```suggestion
    const DWORD runshowcmd[] = { SW_SHOWNORMAL, SW_SHOWMINNOACTIVE, SW_SHOWMAXIMIZED };
```

Reply:
Why? These values are never larger than a byte.

Reply:
Those values are stored in the combobox, and then retrieved later to be stored in the _**DWORD**_ nShowCommand member of the shell link structure.

Reply:
The Microsoft Shortcut specification specifically says that only those 3 SW_ values I used are allowed and I filter them when reading, therefore [it will always fit in a byte](https://github.com/reactos/reactos/pull/5638#discussion_r1312972147).

Reply:
All array members belong to enum of the type `UINT`, however MSDN documents this differently in many places, sometimes `int`, sometimes `DWORD` - all these types are 32-bit anyway. Using `BYTE` here would be an artificial limitation. What matters is not the range of values that the specification dictates, it's the internals, i.e. the way how it's stored in the shortcut. Imagine one day someone would want to extend the range without breaking the file format...

Reply:
> Using `BYTE` here would be an artificial limitation.

There is no BYTE limitation. I am dictating which values are stored in the array. Clearly this is a hardcoded list that will never overflow a BYTE. If Microsoft somehow decides to add new SW_ values that are larger than a BYTE **and** change the .lnk spec to allow them, then yes, it would have to change to a DWORD.